specs/disciplina/educator/api/educator.yaml

```diff
@@ -726,6 +841,30 @@ components:
       allowEmptyValue: true
       schema:
         type: boolean
+    Offset:
+      in: query
+      name: offset
+      description: Pagination parameter. How many items to skip from the beginning.
+      required: false
+      schema:
+        type: integer
+        format: int32
+    Limit:
+      in: query
+      name: limit
+      description: Pagination parameter. Maximum number of items to return.
+      required: false
+      schema:
+        type: integer
+        format: int32
+    Order:
+      in: query
+      name: order
+      description: Return results in ascending or descending order (ascending by default).
```

Comment:
Maybe I just don't see it, but it seems like you didn't specify which field do you sort on,

Reply:
@Martoon-00 sorting parameters will be added after your PR with generic Servant sorting is merged